.circleci/config.yml

```diff
@@ -111,3 +133,12 @@ workflows:
               php-version: [ '5.6', '7.0', '7.1', '7.2', '7.3', '7.4' ]
       - js-tests
       - e2e-tests
+  deploy:
+    jobs:
+      - svn-deploy:
+          context: genesis-svn
+          filters:
+            tags:
+              only: /^\d+\.\d+\.\d+$/
+            branches:
+              ignore: /.*/
```

Comment:
`filters` looks good now.

Another thing you might want here is a `requires` configuration, so that the deploy doesn't happen if the tests fail. You can see how we do that in AB [here](https://github.com/studiopress/atomic-blocks/blob/0d1808b57bcb4caf99ce6de419be1876fac7d02f/.circleci/config.yml#L282-L283).

Reply:
Oh, good idea!